.github/workflows/ci.yml

```diff
@@ -80,7 +83,7 @@ jobs:
     env:
       CIBW_TEST_REQUIRES: "pytest msgpack pyyaml tomli tomli_w"
       CIBW_TEST_COMMAND: "pytest {project}/tests"
-      CIBW_BUILD: "cp38-* cp39-* cp310-* cp311-* cp312-*"
+      CIBW_BUILD: "cp38-* cp39-* cp310-* cp311-* cp312-* cp13-*"
```

Comment:
Usually `pypa/cibuildwheel` minor version takes care of adding and removing EOL and new python versions. I think there is some handling for pre-release but don't remember how.

Could consider moving these to the `pyproject.toml` to make it more easily reproducible and removing the explicit `CIBW_BUILD`?